composer.json

```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
+        "php": "^5.3.3|^7.0|^8.0",
```

Comment:
`5.x` doesn't seem realistic/sensible anymore, but that's up to @webmozart to decide